### PR TITLE
release: 3.0.0-beta25

### PR DESCRIPTION
**Pull request by gregorylegarec**

### CODE REVIEW

**Comment by GoOz** [comment]
There's also the new Sharing button worth mentionning in the Changelog. https://github.com/cozy/cozy-ui/commit/7821f8436f688358537ed0233c6f11d5cb204af6

File: CHANGELOG.md [hunk 1]
```diff
@@ -239,7 +251,8 @@ on desktop or mobile view
 - Everything we did before adopting CHANGELOG…
 
 
-[Unreleased]: https://github.com/cozy/cozy-ui/compare/3.0.0-beta24...HEAD
+[Unreleased]: https://github.com/cozy/cozy-ui/compare/3.0.0-beta25...HEAD
+[3.0.0-beta25]: https://github.com/cozy/cozy-ui/compare/3.0.0-beta23...3.0.0-beta25
```
Comment (GoOz) [inline CHANGELOG.md]: s/beta23/beta24

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - none yet
 
 
+## [3.0.0-beta25] - 2017-05-16
+### Changed
+- Modal now adapts from content width
+
+### Fixed
+- Modal with was irrelevant
```
Comment (GoOz) [inline CHANGELOG.md]: I don't get it. o_o

**Comment by enguerran** [comment]
As it is not released yet, please add the `delete button` from #106 ;)

**Comment by gregorylegarec** [comment]
@enguerran Sorry I've just missed your comment :(

**Pull request merged**